### PR TITLE
Reference property type

### PR DESCRIPTION
**Pull request by fisehara**
Change-type: minor

Signed-off-by: fisehara <harald@balena.io>

### CODE REVIEW

**Comment by fisehara** [comment]
Needs updated sbvr-parser for importing referenceType from sbvr-helper. https://github.com/balena-io-modules/sbvr-parser/pull/38

File: test/referenceType.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+const {
+	TableSpace,
+	term,
+	verb,
+	factType,
+	necessity,
+	referenceType,
+} = require('./sbvr-helper');
+const { Table, attribute } = TableSpace();
+let testterm = term('term');
```
Comment (Page-) [inline test/referenceType.js:11]:
```suggestion
let testTerm = term('term');
```

**Review by Page- — COMMENTED** [review]
LGTM as long as it passes all the testing you've done (and obviously needs the dependent PR merged and updated in package.json here)

**Comment by fisehara** [comment]
@balena-ci I self-certify!

**Pull request merged**